flight/Modules/Actuator/actuator.c

```diff
@@ -338,7 +338,7 @@ static void actuator_task(void* parameters)
 float process_mixer(const int index, const float curve1, const float curve2,
 		ActuatorDesiredData *desired)
 {
-	int8_t *vector = get_mixer_vec(index);
+	typeof(&(*mixerSettings.Mixer1Vector)) vector = get_mixer_vec(index);
```

Comment:
Ugh, torn between having the compiler do smart things for us and having the code be readable. 

```(*vector)[0]``` isn't a pattern we see too often. I suppose I could just put a comment in the code to make it clear what's going on.